gcovr/formats/gcov/read.py

```diff
@@ -579,7 +579,7 @@ def process_datafile(
                 return
     # This exception fails fast
     except GcovrMergeAssertionError as exc:
-        errors.append(str(exc).split("\n"))
+        errors.append(str(exc))
```

Comment:
The errors are joined with indention and therefore we need several entries in the list to get all lines indented:
```suggestion
        errors += str(exc).split("\n")
```